### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by confusdcodr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,11 +25,11 @@ The following elements are created:
...
 * MinCapacity
 * MaxCapacity
-* PrivateSubnetIDs
-* PublicSubnetIDs
-* ConfigBucketName - The S3 bucket to use to grab configuration files
-* GitHubUsername
-* GitHubRepo
+* PrivateSubnetIDs - list of private subnet ids
+* PublicSubnetIDs - list of public subnet ids
+* ConfigBucketName - The S3 bucket to used to grab configuration files
```
Comment (lorengordon) [inline README.md]:
```suggestion
* ConfigBucketName - The S3 bucket used to grab configuration files
```

File: README.md [hunk 2]
```diff
@@ -44,11 +25,11 @@ The following elements are created:
...
+* GitHubUsername - The github username of the salt formula that is going to be getting deployed
+* GitHubRepo - The repository name of the salt formaula that is going to be getting deployed
```
Comment (lorengordon) [inline README.md]:
```suggestion
* GitHubRepo - The repository name of the salt formula that is going to be getting deployed
```

File: README.md [hunk 3]
```diff
@@ -1,37 +1,18 @@
...
-* IAM policy to allow Cloudwatch agent to record metrics
-* IAM policy to allow SSH access to Admin Group
-* IAM policy to allows SSH access to other group
-
-## SNS Template
-The following elements are created:
-* SNS Topic
+## Package CloudFormation Template
+* Packages the local artifacts (local paths) that your AWS CloudFormation template references.
+  * `aws cloudformation package --template-file templates/pwm.compound.json  --s3-bucket <value> > packaged_template.json`
```
Comment (lorengordon) [inline README.md]:
```suggestion
  * `aws cloudformation package --template-file templates/pwm.compound.json --s3-bucket <value> > packaged_template.json`
```

File: README.md [hunk 4]
```diff
@@ -44,11 +25,11 @@ The following elements are created:
...
 * MaxCapacity
-* PrivateSubnetIDs
-* PublicSubnetIDs
-* ConfigBucketName - The S3 bucket to use to grab configuration files
-* GitHubUsername
-* GitHubRepo
+* PrivateSubnetIDs - list of private subnet ids
+* PublicSubnetIDs - list of public subnet ids
+* ConfigBucketName - The S3 bucket to used to grab configuration files
+* GitHubUsername - The github username of the salt formula that is going to be getting deployed
```
Comment (lorengordon) [inline README.md:36]: The explanation highlights that these variable names are a bit awkward. Can press on for now, not worth refactoring.

File: README.md [hunk 5]
```diff
@@ -44,11 +25,11 @@ The following elements are created:
 * DesiredCapacity
 * MinCapacity
 * MaxCapacity
-* PrivateSubnetIDs
-* PublicSubnetIDs
-* ConfigBucketName - The S3 bucket to use to grab configuration files
-* GitHubUsername
-* GitHubRepo
+* PrivateSubnetIDs - list of private subnet ids
```
Comment (lorengordon) [inline README.md]: This is for the EC2 instances, right?
```suggestion
* PrivateSubnetIDs - list of private subnet ids for the EC2 instances
```

File: README.md [hunk 6]
```diff
@@ -44,11 +25,11 @@ The following elements are created:
...
+* PublicSubnetIDs - list of public subnet ids
```
Comment (lorengordon) [inline README.md]: And this is for the... load balancer?
```suggestion
* PublicSubnetIDs - list of public subnet ids for the application load balancer
```

File: README.md [hunk 7]
```diff
@@ -1,37 +1,18 @@
...
+## cfn-pwm
 PWM (https://github.com/pwm-project/pwm) is an open source password self service application for LDAP directories.
 This cloudformation stack creates the infrastructure needed to run PWM behind an application load balancer
 
-# Templates
-## Compound Template
-The root template that forms the basis of the remaining templates
+## Prequisites
+* S3 bucket with the following configuration files
+  * `PwmConfiguration.xml` - the config file used to configure PWM
```
Comment (userhas404d) [inline README.md:7]: I know we have our implementation specific artifacts to fall back on for reference but thoughts on including a bare bones example on how these would be structured (or a link to an external reference if the config is standardized). 

Same question for the other required resources

Comment (confusdcodr) [reply README.md:7]: I do think a future implementation should include a bare bones configuration file. We just aren't there yet.

Comment (lorengordon) [reply README.md:7]: Can we point to the PWM docs as an example? Same for the war files, where can a user get that?

**Review by lorengordon — APPROVED** [review]

**Pull request merged**